bootstrap.py

```diff
@@ -1013,6 +1013,27 @@ def check_rhn_registration():
     return False
 
 
+def validate_connection():
+    """Validate that servers are up """
+    print_generic("Beginning service status and response-time check on port %s " % API_PORT)
+    print_generic("Ensuring services are online and accessible on within %s milliseconds" % options.service_timeout)
+    url = "https://" + options.foreman_fqdn + ":" + API_PORT + "/katello/api/ping"
+    features = get_json(url)['services']
+    svc_list = ["candlepin", "pulp", "pulp_auth", "candlepin_auth", "foreman_tasks"]
```

Comment:
this list will vary between Katello versions, why not iterating over `features.keys()` directly?